ports/timespinner/Timespinner.sh

```diff
@@ -33,14 +32,15 @@ $ESUDO umount "$monofile" || true
 $ESUDO mount "$monofile" "$monodir"
 
 # Setup savedir
-$ESUDO rm -rf ~/.local/share/Timespinner
-mkdir -p ~/.local/share
-ln -sfv "$GAMEDIR/savedata" ~/.local/share/Timespinner
+bind_directories ~/.local/share/Timespinner "$GAMEDIR/savedata"
 
 # Remove all the dependencies in favour of system libs - e.g. the included 
 # newer version of FNA with patcher included
 rm -f System*.dll mscorlib.dll FNA.dll Mono.*.dll
 
+# Remove GOG scripts
+rm -r ./*.sh ./support/*.sh ./support/yad/*.sh
```

Comment:
Will probably need `$ESUDO` for ArkOS.

```suggestion
$ESUDO rm -r ./*.sh ./support/*.sh ./support/yad/*.sh
```